### ISSUE TITLE
It's back: Installation fails due to signature error

### ISSUE THREAD

**Issue by horsteff** [chocolatey/choco#2406]
### What You Are Seeing?
Signature error on chocolateyInstaller.psm1 when installing Chocolatey.

### What is Expected?
No errors; successful installation.

### How Did You Get This To Happen? (Steps to Reproduce)

1. Open an administrative shell
2. Set-ExecutionPolicy AllSigned
3. iex ((New-Object System.Net.WebClient).DownloadString('https://community.chocolatey.org/install.ps1'))

### Output Log
```
Import-Module : File C:\Users\xxx\AppData\Local\Temp\chocolatey\chocoInstall\tools\chocolateyInstall\helpers\chocolateyInstaller.psm1 cannot be loaded.
The contents of file C:\Users\xxx\AppData\Local\Temp\chocolatey\chocoInstall\tools\chocolateyInstall\helpers\chocolateyInstaller.psm1 might have been
changed by an unauthorized user or process, because the hash of the file does not match the hash stored in the digital signature. The script cannot run on
the specified system. For more information, run Get-Help about_Signing..
```

It seems that the UTF-8-BOM was removed from this file with commit de5b853 so installation fails again now.

**Comment by kpozin** [comment]
Seeing this issue, too.

**Comment by corbob** [comment]
This appears to have been fixed in `0.12.1`. I have confirmed this with the following:

1. Set Execution Policy to `AllSigned`.
2. Create a VM Snapshot.
3. Set `$env:ChocolateyVersion = '0.12.0'`.
4. Attempt to install with `iex ((New-Object System.Net.WebClient).DownloadString('https://community.chocolatey.org/install.ps1'))`.
5. Received reported errors.
6. Revert snapshot.
7. Set `$env:ChocolateyVersion = '0.12.1'`.
8. Attempt to install with `iex ((New-Object System.Net.WebClient).DownloadString('https://community.chocolatey.org/install.ps1'))`.
9. Confirmed installed correctly.
10. Revert snapshot.
11. Attempt to install latest with `iex ((New-Object System.Net.WebClient).DownloadString('https://community.chocolatey.org/install.ps1'))`.
12. Confirmed installed correctly.

Looking at other issues, this appears it was due to the file encoding, and resolved as part of #2539.

**Comment by corbob** [comment]
Duplicate of #2539
